controllers/oscmachine_vm_controller.go

```diff
@@ -547,6 +558,9 @@ func reconcileDeleteVm(ctx context.Context, clusterScope *scope.ClusterScope, ma
 		} else {
 			machineSize = 1
 			machineKcpCount = 1
+			machineKwCount = 1
```

Comment:
you can optimize this part of code

Reply:
This part of code is only for unittest. (If not, 
It is never use in real case. i need to avoid clusterScope.ListMachines for unit test because i don't know how to mock it.